docs/source/guide/tutorials/advanced/self_sl.rst

```diff
@@ -0,0 +1,226 @@
+    [*]     - Updated: otx-workspace-CLASSIFICATION/selfsl/model.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/selfsl/data_pipeline.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/deployment.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/hpo_config.yaml
+    [*]     - Updated: otx-workspace-CLASSIFICATION/model_hierarchical.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/model_multilabel.py
+    [*] Update data configuration file to: otx-workspace-CLASSIFICATION/data.yaml
+
+    (otx) ...$ cd ./otx-workspace-CLASSIFICATION
+
+3. To start training we need to call ``otx train`` command in our workspace.
+
+.. note::
+    It is recommended to set ``--save-model-to`` to distinguish between pre-trained and fine-tuned weights or not to overwrite them.
+
+.. code-block::
+
+  (otx) ...$ otx train --save-model-to models/selfsl
+
+The training will return artifacts: ``weights.pth`` and ``label_schema.json`` and we can use this weights to fine-tune the models using target dataset.
+
+
+-----------
+Fine-tuning
+-----------
+
+1. Update our workspace to enable supervised (incremental) learning, which we actually try to do.
+Call the command below from `the root directory` without adding ``--train-type SELFSUPERVISED`` in the command.
+
+.. code-block::
+
+    (otx) ...$ otx build --train-data-roots data/flower_photos --model MobileNet-V3-large-1x
+
+    [*] Workspace Path: otx-workspace-CLASSIFICATION
+    [*] Load Model Template ID: Custom_Image_Classification_MobileNet-V3-large-1x
+    [*] Load Model Name: MobileNet-V3-large-1x
+    [*]     - Updated: otx-workspace-CLASSIFICATION/model.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/data_pipeline.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/deployment.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/hpo_config.yaml
+    [*]     - Updated: otx-workspace-CLASSIFICATION/model_hierarchical.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/model_multilabel.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/compression_config.json
+    [*] Found validation data in your dataset in /home/sungchul/workspace/src/training_extensions/dataset/flower_photos. It'll be used as validation data.
+    [*] Update data configuration file to: otx-workspace-CLASSIFICATION/data.yaml
+
+    (otx) ...$ cd ./otx-workspace-CLASSIFICATION
+
+2. To start training we need to call the below command with adding ``--load-weights`` argument in our workspace.
+
+.. note::
+    It is recommended to set ``--save-model-to`` to distinguish between pre-trained and fine-tuned weights or not to overwrite them.
+
+.. code-block::
+
+  (otx) ...$ otx train --load-weights models/selfsl/weights.pth --save-model-to models/finetune
+
+After these progesses, you can validate, optimize, and export the models described in :doc:`how to train classification models <../base/how_to_train/classification>`.
+
+
+*********************
+Semantic segmentation
```

Comment:
As I can see, the process of training isn't changed compared to classification except different dataset and template. I would rather just wrote few lines about that the Self-SL pretraining for semantic segmentation task is the same,  you just need to change dataset and model template, etc. 

We did the same structure for all common tutorials, we didn't write it for every task

Reply:
Done